### ISSUE TITLE
[$2000] Android - Close account - The scrollbar is too dark

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#14118]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue found when executing PR** https://github.com/Expensify/App/pull/13958

## Action Performed:
1. Launch the app
2. Log in with any account
3. Go to Settings > Security > Close account
4. Add text in the message field until it overflows
5. Confirm the scrollbar is fully visible

## Expected Result:
Scrollbar is fully visible.

## Actual Result:
The scrollbar is too dark

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native

**Version Number:** 1.2.50.13

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/211177802-2cea257c-b949-49bc-bd2d-3e37ac572ce3.mp4

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~014bb6d8d614368b14</li>
        <li>Upwork Job ID: 1613499984023916544</li>
        <li>Last Price Increase: 2023-01-19</li>
    </ul>
</details>

**Comment by flaviadefaria** [comment]
Reproduced, adding the External label.

**Comment by Prince-Mendiratta** [comment]
There is still no fix for this on React Native yet. If it is really required, I can implement a workaround by wrapping the RNTextInput under a ScrollView to get the scrollbar working.

https://github.com/Expensify/App/issues/8131#issuecomment-1300481086 is relevant to decide if this issue should be closed or worked upon.
@parasharrajat might be able to shed more light on this.

**Comment by sobitneupane** [comment]
Rather than a workaround, we prefer the issue to be fixed upstream as discussed in https://github.com/Expensify/App/issues/8131 issue.

**Comment by Prince-Mendiratta** [comment]
I'd love to take an attempt at this but haven't interacted with RN core ever, would appreciate if you could point me in the right direction to start :)

**Comment by marcaaron** [comment]
I think we should maybe have team Margelo look at this one for us. Sorry these conversations tend to escalate so I'm going to lock this issue and discuss it internally before deciding to get community input.

**Comment by marcaaron** [comment]
@JmillsExpensify thoughts on whether we should just close this one or kick it to Margelo? There's a comment [here](https://github.com/Expensify/App/issues/8131#issuecomment-1300481086) where we punted what looks like a similar issue so wanted to get your take first.

**Comment by flaviadefaria** [comment]
Not overdue, just waiting on @JmillsExpensify (comment above) to see how we move forward here.

**Comment by JmillsExpensify** [comment]
Sorry ya'll. I got behind in Github. Hmm, that's a pretty tough one. My first thought was we should close this, but given the great relationship that Margelo has with Meta, it's probably worth giving an upstream fix a shot.

**Comment by marcaaron** [comment]
Ok I will try to summarize this one tomorrow and see what the Margelo guys think.

**Comment by marcaaron** [comment]
**Summary:**

This issue is [the same issue as the one we already closed](https://github.com/Expensify/App/issues/8131). I'm just catching up since this is reaching the 2 week mark, but I think they are _basically_ the same exact issue. 

I think not having the scrollbar here does kind of make our messaging of "platform consistency" a bit confusing. We say all platforms should ideally look and behave similar yet Android has no scrollbar. Then again, this is a really low value UX issue.

It does look like this feature request is being tracked here:

https://react-native.canny.io/feature-requests/p/show-scrollbar-in-android-multiline-textinput

But it was posted back in 2018 so this problem has been floating around in the RN core for 5 years and doesn't seem like there's a ton of interest from the community in fixing it.

**Comment by marcaaron** [comment]
I think we should probably just close this one as a #wontfix for now. If it gets fixed upstream that's cool otherwise feels like we have bigger fish to fry at the moment. If anyone disagrees feel free to reopen.
